docs/configuration.md

```diff
@@ -5,14 +5,34 @@
 | Name                     | Description                                                  |
 | ------------------------ | ------------------------------------------------------------ |
 | kafkaListeners           | Comma-separated list of URIs that we will listen on and the listener names.<br>e.g. PLAINTEXT://localhost:9092,SSL://localhost:9093.<br>If the hostname is not set, the default interface is used. |
+| kafkaProtocolMap         | Comma-separated map of listener name and protocol.<br>e.g. PRIVATE:PLAINTEXT,PRIVATE_SSL:SSL,PUBLIC:PLAINTEXT,PUBLIC_SSL:SSL. |
 | listeners                | Deprecated. `kafkaListeners` is used.                   |
-| kafkaAdvertisedListeners | Listeners published to the ZooKeeper for clients to use.<br>The format is the same as `kafkaListeners`. |
-| kafkaListenerName        | Specify the internal listener name for the broker.<br>The listener name must be contained in the advertisedListeners.<br>This config is used as the listener name in topic lookup. |
+| kafkaAdvertisedListeners | Deprecated. Use kafkaProtocolMap, kafkaListeners and advertisedAddress instead. |
 
 > **NOTE**
 > 
 > Among all configurations, only `kafkaListeners` or `listeners` (deprecated) is required.
 
+To support multiple listeners, you need to specify different listener names in [`advertisedListeners`](https://pulsar.apache.org/docs/en/concepts-multiple-advertised-listeners/#use-multiple-advertised-listeners). Then map the listener name to the proper protocol in `kafkaProtocolMap`.
+
+For example, assuming you're going to listen on port 9092 and 19092 with the `PLAINTEXT` protocol, the associated names are `kafka_internal` and `kafka_external`. Then you need to add the following configurations:
+
+```properties
+kafkaListeners=kafka_internal://localhost:9092,kafka_external://localhost:19092
+kafkaProtocolMap=kafka_internal:PLAINTEXT,kafka_external:PLAINTEXT
+advertisedListeners=pulsar:pulsar://localhost:6650,kafka_internal:pulsar://localhost:9092,kafka_external:pulsar://localhost:19092
+```
+
+In above example,
+- `kafkaListener` is split into multiple tokens by `,`, each token's format is `<listener-name>://<host>:<port>`.
```

Comment:
```suggestion
- `kafkaListener` is split into multiple tokens by `comma (,)`, the format of each token is `<listener-name>://<host>:<port>`.
```

Reply:
For the usage of possessives, see https://developers.google.com/style/possessives
And you can refine the following accordingly.